src/CommonLib/LdapUtils.cs

```diff
@@ -99,7 +92,8 @@ public IAsyncEnumerable<LdapResult<IDirectoryObject>> Query(LdapQueryParameters
             return _connectionPool.Query(queryParameters, cancellationToken);
         }
 
-        public IAsyncEnumerable<LdapResult<IDirectoryObject>> PagedQuery(LdapQueryParameters queryParameters, CancellationToken cancellationToken = new()) {
+        public IAsyncEnumerable<LdapResult<IDirectoryObject>> PagedQuery(LdapQueryParameters queryParameters,
+            CancellationToken cancellationToken = new()) {
```

Comment:
Are we doing anything with the cancellation token?

Reply:
Yes it gets passed down to the lower level functions to cancel out of queries if needed